src/db/migrations/20190821154724-create-trip.js

```diff
@@ -21,7 +21,7 @@ module.exports = {
         type: Sequelize.ENUM('one-way', 'round-trip', 'multi-city'),
         allowNull: false
       },
-      from: {
+      origin: {
```

Comment:
Did we agree on changing this to origin?

Reply:
The words `origin` and `destination` are the perfect concise instruments for trips and travels.
The word `from` is  ambiguous and can be confusing.

Reply:
Not a problem. I just wanted to be clear because when Olisa suggested it during the meeting, the team did not second. When changes that affect everyone is made, it's a good idea that we all agree beforehand.

Reply:
function duringMeeting(use_origin) {
    team_members_opinion = rejected;
   mitchell != team_members
}